debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -773,31 +773,31 @@ protected void assertRecordSchemaAndValues(List<SchemaAndValueField> expectedSch
     protected void assertRecordOffsetAndSnapshotSource(SourceRecord record, boolean shouldBeSnapshot, boolean shouldBeLastSnapshotRecord) {
         Map<String, ?> offset = record.sourceOffset();
         assertNotNull(offset.get(SourceInfo.TXID_KEY));
-        assertNotNull(offset.get(SourceInfo.TIMESTAMP_KEY));
+        assertNotNull(offset.get(SourceInfo.TIMESTAMP_USEC_KEY));
         assertNotNull(offset.get(SourceInfo.LSN_KEY));
         Object snapshot = offset.get(SourceInfo.SNAPSHOT_KEY);
-        Object lastSnapshotRecord = offset.get(SourceInfo.LAST_SNAPSHOT_RECORD_KEY);
+//        Object lastSnapshotRecord = offset.get(SourceInfo.LAST_SNAPSHOT_RECORD_KEY);
```

Comment:
Why are those commented out? Is it some left-over?

Reply:
@gunnarmorling This is intentional, the test will be uncommented (and updated) with switch to three-state snapshot

Reply:
So when would we do this (sorry, don't remember the outcome of our discussion). Shouldn't this be part of this PR here then?

Reply:
@gunnarmorling IIRC We agreed we will do all but snapshot 3state to get this version out soon and the next release will contain updated snapshot.